README.md

```diff
@@ -480,6 +480,16 @@ Options:
                         separated
 ~~~
 
+# Additional Notes
+
+## FIPS support
+
+On systems with FIPS enabled (where `/proc/sys/crypto/fips_enabled` == `1`), algorithms such as MD5 are disallowed.
```

Comment:
I think this should be '/proc/sys/crypto/fips_enabled == 1' (but using backticks, of course)